js/core/src/Blocks.svelte

```diff
@@ -623,6 +623,33 @@
 
 			/* eslint-disable complexity */
 			function handle_status_update(message: StatusMessage): void {
+				if (message.code === "validation_error") {
+					const dep = dependencies.find((dep) => dep.id === message.fn_index);
+					const validation_error_data = [];
+
+					message.message.forEach((message, i) => {
+						if (message.is_valid) {
+							return;
+						}
+						validation_error_data.push({
+							id: dep.inputs[i],
+							prop: "validation_error",
+							value: message.message
+						});
+					});
+
```

Comment:
We should skip the rest of the block if the validators are all true, it shouldn't happen but better to be defensive.